### PR TITLE
HTML anchor tag to markdown in 2.1

### PR DESCRIPTION
**Pull request by jignesh-baldha**
## This PR is a:

- [ ] New topic
- [X] Content update
- [X] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will HTML achor to Markdown.

### CODE REVIEW

File: guides/v2.1/cloud/reference/git-integration.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ If you need help understand Git, you can review the following resources:
 *	[Git video](https://www.youtube.com/watch?v=8KCQe9Pm1kg){:target="\_blank"} with DevForge to understand how people use the repo and commands with a fun story
 
 ## Git CLI and clients {#clients}
-You can interact with Git using [CLI commands](https://git-scm.com/documentation){:target="\_blank"} or using a Git client. Git provides a <a href="https://git-scm.com/downloads" target="\_blank">Git client</a> option, or you can use other clients such as installed on your computer to be able to interact with {{site.data.var.ece}}.
+You can interact with Git using [CLI commands](https://git-scm.com/documentation){:target="\_blank"} or using a Git client. Git provides a [Git client](https://git-scm.com/downloads" target="\_blank) option, or you can use other clients such as installed on your computer to be able to interact with {{site.data.var.ece}}.
```
Comment (shrielenee) [inline guides/v2.1/cloud/reference/git-integration.md]: This isn't the correct way to use a blank target in markdown - `{:target="_blank"}`

File: guides/v2.1/coding-standards/docblock-standard-javascript.md [hunk 2]
```diff
@@ -15,9 +15,9 @@ functional_areas:
...
 
 To add {% glossarytooltip 312b4baf-15f7-4968-944e-c814d53de218 %}JavaScript{% endglossarytooltip %} code inline documentation, follow these guidelines. Some parts of Magento code may not comply with this standard, but we are working to gradually improve this. Following these standard is optional for 3rd-party Magento developers, but will help to create consistent, clean, and easy to read inline documentation.
-This standard are a subset of <a href="https://google.github.io/styleguide/javascriptguide.xml" target="_blank">Google JavaScript Style Guide</a> regulations.
+This standard are a subset of [Google JavaScript Style Guide](https://google.github.io/styleguide/javascriptguide.xml" target="_blank) regulations.
 
-<p>Use <a href="http://www.ietf.org/rfc/rfc2119.txt" target="_blank">RFC 2119</a> to interpret the "must," "must not," "required," "shall," "shall not," "should," "should not," "recommended," "may," and "optional" keywords.</p>
+<p>Use [RFC 2119](http://www.ietf.org/rfc/rfc2119.txt" target="_blank) to interpret the "must," "must not," "required," "shall," "shall not," "should," "should not," "recommended," "may," and "optional" keywords.</p>
```
Comment (shrielenee) [inline guides/v2.1/coding-standards/docblock-standard-javascript.md]: Markdown targets don't work like this. Try `{:target="_blank"}` after the link in parentheses.

File: guides/v2.1/comp-mgr/prereq/prereq_cron.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ To enable us to update or upgrade your system, you must configure two cron jobs.
 
 The cron jobs schedule tasks for the Setup Wizard and for the updater application. These applications work together to install, update, and upgrade the Magento application and components.
 
-Enable the cron jobs as <a href="http://ss64.com/bash/crontab.html" target="_blank">crontabs</a> for the [Magento file system owner]({{ page.baseurl }}/install-gde/prereq/file-sys-perms-over.html) because that user runs the updater application for the Web Setup Wizard. 
+Enable the cron jobs as [crontabs](http://ss64.com/bash/crontab.html" target="_blank) for the [Magento file system owner]({{ page.baseurl }}/install-gde/prereq/file-sys-perms-over.html) because that user runs the updater application for the Web Setup Wizard.
```
Comment (shrielenee) [inline guides/v2.1/comp-mgr/prereq/prereq_cron.md]: Wrong target format

File: guides/v2.1/config-guide/cli/config-cli-subcommands-i18n.md [hunk 4]
```diff
@@ -255,7 +255,7 @@ If the Magento application cannot find word or phrase in the `en_GB` package, it
 1.  `<vendorname>/en_ca_package`
 1.  `<vendorname>/en_us_package`
 
-Specifying all inheritances between the language packages might result in creating circular inheritance chains. Use <a href="{{ site.mage2000url }}dev/tests/static/testsuite/Magento/Test/Integrity/App/Language/CircularDependencyTest.php" target="\_blank">Magento\Test\Integrity\App\Language\CircularDependencyTest</a> test to locate and fix such chains.
+Specifying all inheritances between the language packages might result in creating circular inheritance chains. Use [Magento\Test\Integrity\App\Language\CircularDependencyTest]({{ site.mage2000url }}dev/tests/static/testsuite/Magento/Test/Integrity/App/Language/CircularDependencyTest.php" target="\_blank) test to locate and fix such chains.
```
Comment (shrielenee) [inline guides/v2.1/config-guide/cli/config-cli-subcommands-i18n.md]: Wrong target formatting

**Comment by shrielenee** [comment]
I didn't review all the files, but I did comment on a few things I noticed that needed revision. Also make sure you fix the conflicting files. Thanks!

**Comment by jignesh-baldha** [comment]
@shrielenee All those conflicts are due to PR #2672, In this PR converted HTML anchor tags to markdown without target links. That seems incorrect, could you please review #2672 PR.

Also I will close this PR and will create another PR will changes you requested.

**Comment by shrielenee** [comment]
PR #2672 has already been merged @jignesh-baldha. You should be able to resolve your conflicts by updating your local repo with merged devdocs changes. Let me know how you'd like to proceed!

**Comment by jignesh-baldha** [comment]
@shrielenee I would like to proceed with new PR.

**Comment by shrielenee** [comment]
No problem @jignesh-baldha - I will close this PR for now. If you would like to open it please feel free.

**Pull request closed without merging**